com.unity.render-pipelines.high-definition/Runtime/RenderPipeline/HDRenderPipeline.cs

```diff
@@ -4122,6 +4112,55 @@ void RenderTransparencyOverdraw(CullingResults cull, HDCamera hdCamera, Scriptab
             }
         }
 
+        struct FullScreenDebugParameters
+        {
+            public RendererListDesc rendererList;
+            public FrameSettings frameSettings;
+        }
+
+        FullScreenDebugParameters PrepareFullScreenDebugParameters(HDCamera hdCamera, CullingResults cull)
+        {
+            var parameters = new FullScreenDebugParameters();
+
+            parameters.rendererList = CreateOpaqueRendererListDesc(cull, hdCamera.camera, m_FullScreenDebugPassNames, renderQueueRange: RenderQueueRange.all);
+            parameters.frameSettings = hdCamera.frameSettings;
+
+            return parameters;
+        }
+
+        void RenderFullScreenDebug( FullScreenDebugParameters   parameters,
```

Comment:
This function should be static otherwise when called from a render graph pass, it will capture "this" and generate GCAlloc.
To check this, you can hover the mouse over the => of the lambda calling this and check that it does not capture anything.